core/include/scipp/core/string.h

```diff
@@ -92,9 +94,9 @@ std::string array_to_string(const T &arr,
     return std::string("[]");
   std::string s = "[";
   for (scipp::index i = 0; i < scipp::size(arr); ++i) {
-    if (i == 2 && size > 4) {
+    if (i == 2 && size > 6) {
```

Comment:
Didn't actually mean to commit this change. Should I revert it?

Reply:
Yes.